openmc/deplete/results.py

```diff
@@ -91,7 +93,13 @@ def from_hdf5(cls, filename):
         )
         return cls(filename)
 
-    def get_atoms(self, mat, nuc, nuc_units="atoms", time_units="s"):
+    def get_atoms(
+        self,
+        mat: Material,
```

Comment:
```suggestion
        mat: Union[Material, str],
```

Reply:
Ah sorry I got this in the get reaction rates and missed it here